src/app/components/Document/index.jsx

```diff
@@ -27,16 +60,11 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
         {title}
         {links}
         {styleTags}
+        {ampScript}
       </head>
       <body>
-        {/* eslint-disable react/no-danger */
```

Comment:
Can this be added back?

Reply:
It's now added further up the file and the way we've disabled it cascades down the file, happy to change these disables to be inline and therefore more explicit